app/uk/gov/hmrc/minorentityidentificationfrontend/testonly/forms/TestCreateJourneyForm.scala

```diff
@@ -77,17 +80,19 @@ object TestCreateJourneyForm {
       signOutUrl -> text.verifying(signOutUrlEmpty),
       accessibilityUrl -> text.verifying(accessibilityUrlEmpty),
       businessVerificationCheck -> boolean,
-      regime -> text.verifying(regimeEmpty)
-    )((continueUrl, serviceName, deskProServiceId, signOutUrl, accessibilityUrl, businessVerificationCheck, regime) =>
-      JourneyConfig.apply(continueUrl, PageConfig(serviceName, deskProServiceId, signOutUrl, accessibilityUrl), businessEntity, businessVerificationCheck, regime)
+      regime -> text.verifying(regimeEmpty),
+      welshServiceName -> optText
+    )((continueUrl, serviceName, deskProServiceId, signOutUrl, accessibilityUrl, businessVerificationCheck, regime, welshServiceName) =>
+      JourneyConfig.apply(continueUrl, PageConfig(serviceName, deskProServiceId, welshServiceName, signOutUrl, accessibilityUrl), businessEntity, businessVerificationCheck, regime)
     )(journeyConfig =>
       Some(journeyConfig.continueUrl,
         journeyConfig.pageConfig.optServiceName,
         journeyConfig.pageConfig.deskProServiceId,
         journeyConfig.pageConfig.signOutUrl,
         journeyConfig.pageConfig.accessibilityUrl,
         journeyConfig.businessVerificationCheck,
-        journeyConfig.regime
+        journeyConfig.regime,
+        if(journeyConfig.pageConfig.labels.isDefined) journeyConfig.pageConfig.labels.map(_.welshServiceName) else None
```

Comment:
you can use this:
        journeyConfig.pageConfig.labels.map(_.welshServiceName)
if you want to.